packages/compiler/core/parser.ts

```diff
@@ -26,6 +26,7 @@ import {
   DirectiveExpressionNode,
```

Comment:
Believe that might require a change to `spec.enum.html`? @nguerrera

Reply:
thanks, updated